Sources/Components/Style/WKColor.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+import SwiftUI
+
+public enum WKColor {
```

Comment:
Minor but enums without cases always feel a bit weird to me. I'm open to keeping it if there's consensus though.

Reply:
I can understand your viewpoint – there are times I also go back and forth between approaches. In the case of colors, I've generally always found the caseless enum approach has a nice balance in use.

- Namespacing: I mostly treat the enum as just a way to namespace the colors (which are just static properties that don't need to be computed like with our fonts). By going with an enum type, we can ensure the developer doesn't accidentally instantiate a `Color` as they would if it were a struct or class.

```swift
enum Color { ... }
❌ let color = Color()

struct Color { ... }
✅ let color = Color()

class Color { ... }
✅ let color = Color()
```



- More concise: The static properties on an enum approach is much more concise, in definition and in use, than making cases for each color.

```swift
enum Colors {
	static let systemBlack = UIColor(...)
	static let systemRed = UIColor(...)
}

textView.textColor = Colors.systemBlack

vs.

enum MyColor {
    case systemBlack
    case systemRed

    var rawValue: UIColor {
        switch self {
        case .systemRed: return UIColor(...)
        case .systemBlack: return UIColor(...)
        }
    }
}

textView.textColor = Colors.systemBlack.rawValue
```

We could certainly write something similar to `WKFont`'s `static func for(...)` approach and use cases for colors as well if my approach here is unpopular, but I think we'd benefit from the more concise, namespace-only approach like above because our colors don't have the same computed requirements as our fonts. Totally open to discussion if there are different ideas here.

Reply:
👍 Makes sense, thanks for the examples! I'm good with keeping them as case-less enums.